src/core/declarativewebcontainer.h

```diff
@@ -130,11 +132,13 @@ class DeclarativeWebContainer : public QWindow, public QQmlParserStatus, protect
     bool activatePage(const Tab& tab, bool force = false, int parentId = 0);
     int findParentTabId(int tabId) const;
 
-    Q_INVOKABLE void load(QString url = "", QString title = "", bool force = false);
+    Q_INVOKABLE void load(QString url = "", bool force = false);
```

Comment:
Just more of a style thing but everywhere else strings have been converted to references but not this one

Reply:
Heck... I'll replace them all. :)

Reply:
All fixed now and changed amended to the style fix commit.